mailbox/elasticsearch/src/test/java/org/apache/james/mailbox/elasticsearch/events/ElasticSearchListeningMessageSearchIndexTest.java

```diff
@@ -18,6 +18,7 @@
  ****************************************************************/
```

Comment:
The test you added should be on AbstractMessageSearchIndexTest. We want to ensure the different implementation got the same behaviour.

What I can not enforce while simply reading your code.